### PR TITLE
fix: hire an exterminator

### PR DESCRIPTION
**Pull request by eclairevoyant**
heard you had a rat problem, and do i have a solution for you :x: :rat: :x:

### CODE REVIEW

**Comment by NotAShelf** [comment]
🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀 🐀

**Comment by NotAShelf** [comment]
the rats shot the exterminator :sob:

File: README.md [hunk 1]
```diff
@@ -14,7 +14,6 @@ There are several packages exposed by this flake. Each directory in `pkgs` conta
 | headscale-ui       |             A web frontend for the headscale Tailscale-compatible coordination server              |
 | mastodon-bird-ui   |                         Mastodon web UI, but strongly inspired by Twitter.                         |
 | mov-cli            |                       A cli tool to browse and watch Movies/Shows/TV/Sports                        |
-| rat                | Linux shell port of the horizontally spinning rat meme, complete with soundtrack and spin counter. |
 | reposilite-bin     |                         A derivation for the reposilite maven repository.                          |
 | rofi-calc-wayland  |          A wayland patched version of [rofi-calc](https://github.com/svenstaro/rofi-calc)          |
```
Comment (eclairevoyant) [inline README.md:18]: ❌ 🐀 ❌

File: README.md [hunk 2]
```diff
@@ -14,7 +14,6 @@ There are several packages exposed by this flake. Each directory in `pkgs` conta
 | headscale-ui       |             A web frontend for the headscale Tailscale-compatible coordination server              |
 | mastodon-bird-ui   |                         Mastodon web UI, but strongly inspired by Twitter.                         |
 | mov-cli            |                       A cli tool to browse and watch Movies/Shows/TV/Sports                        |
-| rat                | Linux shell port of the horizontally spinning rat meme, complete with soundtrack and spin counter. |
 | reposilite-bin     |                         A derivation for the reposilite maven repository.                          |
```
Comment (eclairevoyant) [inline README.md:17]: ```suggestion\n❌ 🐀 ❌\n```

**Pull request closed without merging**